runtime-sofa-boot-starter/src/main/java/com/alipay/sofa/runtime/spring/ServiceAnnotationBeanPostProcessor.java

```diff
@@ -35,52 +32,50 @@
 import com.alipay.sofa.runtime.annotation.PlaceHolderBinder;
 import com.alipay.sofa.runtime.api.ServiceRuntimeException;
 import com.alipay.sofa.runtime.api.annotation.SofaReference;
-import com.alipay.sofa.runtime.api.annotation.SofaService;
-import com.alipay.sofa.runtime.api.annotation.SofaServiceBinding;
 import com.alipay.sofa.runtime.api.binding.BindingType;
-import com.alipay.sofa.runtime.constants.SofaRuntimeFrameworkConstants;
 import com.alipay.sofa.runtime.model.InterfaceMode;
 import com.alipay.sofa.runtime.service.binding.JvmBinding;
 import com.alipay.sofa.runtime.service.component.Reference;
-import com.alipay.sofa.runtime.service.component.Service;
-import com.alipay.sofa.runtime.service.component.ServiceComponent;
 import com.alipay.sofa.runtime.service.component.impl.ReferenceImpl;
-import com.alipay.sofa.runtime.service.component.impl.ServiceImpl;
 import com.alipay.sofa.runtime.service.helper.ReferenceRegisterHelper;
 import com.alipay.sofa.runtime.spi.binding.Binding;
 import com.alipay.sofa.runtime.spi.binding.BindingAdapterFactory;
-import com.alipay.sofa.runtime.spi.component.ComponentInfo;
-import com.alipay.sofa.runtime.spi.component.DefaultImplementation;
-import com.alipay.sofa.runtime.spi.component.Implementation;
 import com.alipay.sofa.runtime.spi.component.SofaRuntimeContext;
 import com.alipay.sofa.runtime.spi.service.BindingConverter;
 import com.alipay.sofa.runtime.spi.service.BindingConverterContext;
 import com.alipay.sofa.runtime.spi.service.BindingConverterFactory;
 
 /**
+ * Responsible to inject field annotated by @SofaReference and
+ * invoke setXX method annotated by @SofaReference
+ *
  * @author xuanbei 18/5/9
  */
-public class ServiceAnnotationBeanPostProcessor implements BeanPostProcessor, Ordered,
-                                               ApplicationContextAware {
+public class ServiceAnnotationBeanPostProcessor implements BeanPostProcessor, Ordered {
```

Comment:
Should we rename this class to make it more clear?

Reply:
ok